deployment/aks-periscope.yaml

```diff
@@ -114,15 +116,15 @@ metadata:
   name: containerlogs-config
   namespace: aks-periscope
 data:
-  DIAGNOSTIC_CONTAINERLOGS_LIST: kube-system
```

Comment:
💡 Why is this `kube-system` remove? will this have any impact on existing functionality this tool provide?

Reply:
I changed it because for arc customizations, we would need to get container logs and kube-objects from the azure-arc namespace. So I removed kube-system as the default namespace and changed it to azure-arc. The same thing applies to kube-object config. Of course, we can have azure-arc and kube-system as both of the default namespaces. This was something I was going to ask during the next sync with my team and your team, how we plan on going about setting the default namespace in the yaml. I believe Safeer was also looking into the possibility of using a Helm Chart for periscope.

Reply:
These values can be overwritten by CLI, we would not need these change. @sophsoph321, can you please revert this change? 

---
In reply to: [626987398](https://github.com/Azure/aks-periscope/pull/51#discussion_r626987398) [](ancestors = 626987398)